packages/twenty-server/src/engine/core-modules/graphql/hooks/use-graphql-error-handler.hook.ts

```diff
@@ -72,7 +72,7 @@ export const useGraphQLErrorHandlerHook = <
                   : generateGraphQLErrorFromError(error);
               });
 
-              const errorsToCapture = originalErrors.reduce<BaseGraphQLError[]>(
+              const errorsToCapture = result.errors.reduce<BaseGraphQLError[]>(
```

Comment:
this looks wrong, could you justify it? I remember we tried this change earlier but put it back